spec/mongo/server/connection_pool/populator_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe Mongo::Server::ConnectionPoolPopulator do
+  let(:server) do
+    authorized_client.cluster.next_primary
+  end
+
+  let(:pool) do
+    server.pool
+  end
+
+  let(:populator) do
+    described_class.new(pool, pool.options)
```

Comment:
Should we be doing some kind of clean up for the populator after the test is run?

Reply:
This is probably a good idea.